### PR TITLE
Fix pycparser installation

### PR DESCRIPTION
**Pull request by qiluo-msft**
Fix https://github.com/Azure/sonic-mgmt/issues/651

### CODE REVIEW

**Comment by qiluo-msft** [comment]
Retest this please

**Comment by qiluo-msft** [comment]
Retest this please

File: dockers/docker-sonic-mgmt/Dockerfile.j2 [hunk 1]
```diff
@@ -29,7 +29,8 @@ RUN apt-get install -y default-jre
 RUN apt-get install -y rsyslog psmisc
 
 # Remove cffi 1.5.2, will install 1.10.0 by pip later
-RUN apt-get remove -y python-cffi python-cffi-backend
+# Remove pycparser 2.14, will install >=2.17 by pip later
+RUN apt-get remove -y python-cffi python-cffi-backend python-ply python-pycparser
```
Comment (jleveque) [inline dockers/docker-sonic-mgmt/Dockerfile.j2]: Why not separate into two `apt-get remove` calls for posterity's sake?

```
# Remove cffi 1.5.2, will install 1.10.0 by pip later
RUN apt-get remove -y python-cffi python-cffi-backend

# Remove pycparser 2.14, will install >=2.17 by pip later
RUN apt-get remove -y python-ply python-pycparser
```

**Comment by qiluo-msft** [comment]
Retest this please

**Comment by qiluo-msft** [comment]
Retest this please

**Pull request merged**